Sources/OTel/OTelAPI/OTel+Bootstrap.swift

```diff
@@ -124,32 +124,57 @@ extension OTel {
         var services: [Service] = []
 
         if configuration.logs.enabled {
-            try services.append(bootstrapLogs(configuration: configuration))
+            try services.append(bootstrapLogs(configuration: configuration, logger: logger))
         }
         if configuration.metrics.enabled {
-            try services.append(bootstrapMetrics(configuration: configuration))
+            try services.append(bootstrapMetrics(configuration: configuration, logger: logger))
         }
         if configuration.traces.enabled {
-            try services.append(bootstrapTraces(configuration: configuration))
+            try services.append(bootstrapTraces(configuration: configuration, logger: logger))
         }
 
         return ServiceGroup(services: services, logger: logger)
     }
 
-    internal static func bootstrapTraces(configuration: OTel.Configuration) throws -> some Service {
+    internal static func bootstrapTraces(configuration: OTel.Configuration, logger: Logger) throws -> some Service {
         let backend = try makeTracingBackend(configuration: configuration)
         InstrumentationSystem.bootstrap(backend.factory)
         return backend.service
     }
 
-    internal static func bootstrapMetrics(configuration: OTel.Configuration) throws -> some Service {
+    internal static func bootstrapMetrics(configuration: OTel.Configuration, logger: Logger) throws -> some Service {
         let backend = try makeMetricsBackend(configuration: configuration)
         MetricsSystem.bootstrap(backend.factory)
         return backend.service
     }
 
-    internal static func bootstrapLogs(configuration: OTel.Configuration) throws -> some Service {
+    internal static func bootstrapLogs(configuration: OTel.Configuration, logger: Logger) throws -> some Service {
         let backend = try makeLoggingBackend(configuration: configuration)
+        if configuration.logs.exporter.backing != .console {
+            logger.info(
+                """
+
+                Bootstapping logging system
+                ---
+                Only diagnostic logging will use the console logger.
+
+                If you require console logging for local development, use the
+                console logs exporter, which can be enabled using the
+                following configuration:
+
+                    config.logs.exporter = .console
+
+                Or, run your process with the following environment variable:
+
+                    OTEL_LOGS_EXPORTER=console
+
+                If you require logs to go to both the console and another
+                exporter, manually bootstrap the logging subsystem with a
+                multiplex log hander and the result of `makeLoggingBackend`.
```

Comment:
Could we add the code snippet that does that? I realize it's a slightly more advanced use case, but it'll likely be cargo-culted rather than written from scratch most of the time anyway, so we might as well offer it here.

Reply:
However, I'm disinclined to include the multiplex code snippet, since the docs for `makeLoggingBackend` contain the full snippet for manual bootstrap and even has a dedicated section for multiplexing:

https://swiftpackageindex.com/swift-otel/swift-otel/main/documentation/otel/otel/makeloggingbackend(configuration:)#Multiplex-with-other-logging-backends